modules/oadp-ssec-encrypted-backups.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-ssec-encrypted-backups_{context}"]
+= Creating a OADP SSE-C encryption key for additional data security
+
+{aws-first} S3 now applies server-side encryption with Amazon S3 managed keys (SSE-S3) as the base level of encryption for every bucket in Amazon S3.
+
+Note that {oadp-first} encrypts data by using SSL/TLS, HTTPS, and uses the `velero-repo-credentials` secret when transferring the data from the cluster to storage.
+
+Therefore, it is important to include an additional layer of encryption to prevent backed-up data from being exposed if you lose your AWS credentials or they are stolen.
+
+The _velero-plugin-for-aws_ provides several additional encryption methods. You should review its configuration options and consider implementing additional encryption.
+
+Server-side encryption with customer-provided keys (SSE-C) gives you the ability to store your own encryption keys. This provides additional security if your AWS credentials become exposed.
+
+[WARNING]
+====
+It is important to secure cryptographic keys in a secure and safe manner. Encrypted data and backups will not be recoverable in the absence of the encryption key.
+====
+
+.Prerequisites and known issues
+
+You might receive `credentials not found` errors. To avoid this issue you can create a workaround by adding a `volumeSnapshot` configuration to your `DPA` without specifying a credential as in the following example:
```

Comment:
```suggestion
You might receive `credentials not found` errors. To avoid this issue, you can create a workaround by adding a `volumeSnapshot` configuration to your `DPA` without specifying a credential as in the following example:
```